File: cfgs/java_cfgs_foo/COUNT_NEGATIVE_NUMBERS_IN_A_COLUMN_WISE_ROW_WISE_SORTED_MATRIX_1.java
Nodes
1;special;;;
2;statement;int count = 0;14;
3;statement;int i = 0;15;
4;statement;int j = foo_1 - 1;16;
5;conditional;(foo_1 >= 0) && (foo_2 < foo_3);17;
6;conditional;foo_1[foo_2][foo_3] < 0;18;
7;statement;count += foo_1 + 1;19;
8;statement;i += 1;20;
9;statement;j -= 1;22;
10;statement;return foo_1;24;
11;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;10;6;
5;7;9;
6;8;
7;5;
8;5;
9;11;
10;

File: cfgs/pythoncfgs/COUNT_NEGATIVE_NUMBERS_IN_A_COLUMN_WISE_ROW_WISE_SORTED_MATRIX_1.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = foo_2 - 1;10;
4;conditional;foo_1 >= 0 and foo_2 < foo_3;11;
5;statement;return foo_1;17;
6;conditional;foo_1[foo_2][foo_3] < 0;12;
7;statement;foo_1 -= 1;16;
8;statement;foo_1 += foo_2 + 1;13;
9;statement;foo_1 += 1;14;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;10;
5;7;8;
6;4;
7;9;
8;4;
9;
